### PR TITLE
Update anchor default styles

### LINKED ISSUE #1072: Clean up anchor <a> style so by default it doesn't have underline
Related to https://github.com/mozilla/network-pulse/pull/1054#discussion_r285205354

1. figure out where `a { text-decoration: underline; }` rule in the compiled css file is coming from
2. make sure by default anchors don't have underline
3. other other `text-decoration` related overrides in the codebase accordingly.

### PR DESCRIPTION
**Pull request by youriwims**
Closes  #1072 

_*Updated `@mixin link-text-decoration` to `text-decoration: none` default, but underline on hover_

### CODE REVIEW

File: scss/overrides/_variables.scss [hunk 1]
```diff
@@ -21,7 +21,6 @@ $form-error-color: #d24735;
 $body-color: $black;
 $link-color: inherit;
 $link-hover-color: darken($mobile-blue, 15%);
-$link-decoration: underline;
```
Comment (mmmavis) [inline scss/overrides/_variables.scss]: let's keep this

Comment (mmmavis) [inline scss/overrides/_variables.scss]: let's keep this

File: scss/mixin.scss [hunk 2]
```diff
@@ -18,7 +18,7 @@
   }
 }
 
-@mixin link-text-decoration($normal-state: $link-decoration, $hover-state: $link-decoration) {
+@mixin link-text-decoration($normal-state: none, $hover-state: underline) {
```
Comment (mmmavis) [inline scss/mixin.scss]: `@mixin link-text-decoration($normal-state: none, $hover-state: $link-decoration) { ... `

**Comment by mmmavis** [comment]
@youriwims do you know if we want ALL anchors to 1) not have any `text-decoration` by default 2) and have underline only on hover? If so, we might be able to clean up the SCSS further.

**Comment by youriwims** [comment]
@mmmavis if we're aligning with foundation styles, then yes. @natalieworth can you confirm, pls?

**Comment by youriwims** [comment]
@natalieworth ⬆️

**Comment by youriwims** [comment]
@mmmavis I'm going to file a ticket for enhancement to see if we can clean up `<a>` styles further so this can be landed. (will update this comment with ticket#)

**Pull request merged**